openerp/addons/base/ir/ir_translation.py

```diff
@@ -416,7 +416,8 @@ def translate_fields(self, cr, uid, model, id, field=None, context=None):
         langs_ids = self.pool.get('res.lang').search(cr, uid, [('code', '!=', 'en_US')], context=context)
         if not langs_ids:
             raise osv.except_osv(_('Error'), _("Translation features are unavailable until you install an extra OpenERP translation."))
-        langs = [lg.code for lg in self.pool.get('res.lang').browse(cr, uid, langs_ids, context=context)]
+        langs = [lg.code for lg in self.pool.get('res.lang').browse(
+            cr, uid, langs_ids, context=context) if lg.translatable]
```

Comment:
Any reason you don't add this criterium to the domain in l.416 instead of the list filter?